sql/item_mods.sql

```diff
@@ -1167,6 +1606,41 @@ INSERT INTO `item_mods` VALUES (10639,114,13);
 INSERT INTO `item_mods` VALUES (10640,1,28);
 INSERT INTO `item_mods` VALUES (10640,169,12);
 INSERT INTO `item_mods` VALUES (10640,374,4);
+INSERT INTO `item_mods` VALUES (10641,1,23); -- UGOL SOLERETS, DEF:23 STR+6 VIT+6 AGI-3 [Element:Air]+10 [Element:Earth]+10 Attack+6
+INSERT INTO `item_mods` VALUES (10641,8,6); -- STR
+INSERT INTO `item_mods` VALUES (10641,10,6); -- VIT
+INSERT INTO `item_mods` VALUES (10641,11,-3); -- AGI
+INSERT INTO `item_mods` VALUES (10641,56,10); -- ELEWIND
+INSERT INTO `item_mods` VALUES (10641,57,10); -- ELEEARTH
+INSERT INTO `item_mods` VALUES (10641,23,6); -- ATT
+INSERT INTO `item_mods` VALUES (10642,1,24); -- MAVROS SOLERETS, DEF:24 STR+7 VIT+7 AGI-4 [Element:Air]+15 [Element:Earth]+15 Attack+7
+INSERT INTO `item_mods` VALUES (10642,8,7); -- STR
+INSERT INTO `item_mods` VALUES (10642,10,7); -- VIT
+INSERT INTO `item_mods` VALUES (10642,11,-4); -- AGI
+INSERT INTO `item_mods` VALUES (10642,56,15); -- ELEWIND
+INSERT INTO `item_mods` VALUES (10642,57,15); -- ELEEARTH
+INSERT INTO `item_mods` VALUES (10642,23,7); -- ATT
+INSERT INTO `item_mods` VALUES (10643,1,26); -- URJA LEDELSENS, DEF:26 AGI+5 Attack+5 Enhances "Resist Sleep" effect
+INSERT INTO `item_mods` VALUES (10643,11,5); -- AGI
+INSERT INTO `item_mods` VALUES (10643,23,5); -- ATT
+INSERT INTO `item_mods` VALUES (10643,240,1); -- EN RESSLEEP
+INSERT INTO `item_mods` VALUES (10644,1,27); -- STHIRA LEDELSENS, DEF:27 AGI+6 Attack+6 Enhances "Resist Sleep" effect
+INSERT INTO `item_mods` VALUES (10644,11,6); -- AGI
+INSERT INTO `item_mods` VALUES (10644,23,6); -- ATT
+INSERT INTO `item_mods` VALUES (10644,240,1); -- EN RESSLEEP
+INSERT INTO `item_mods` VALUES (10645,1,18); -- SPOLIA PIGACHES, DEF:18 MP+40 "Magic Atk. Bonus"+4 Enmity-6
+INSERT INTO `item_mods` VALUES (10645,5,40); -- MP
+INSERT INTO `item_mods` VALUES (10645,28,4); -- MAB
+INSERT INTO `item_mods` VALUES (10645,27,-6); -- ENMITY
+INSERT INTO `item_mods` VALUES (10646,1,19); -- OPIMA PIGACHES, DEF:19 MP+45 "Magic Atk. Bonus"+5 Enmity-7
+INSERT INTO `item_mods` VALUES (10646,5,45); -- MP
+INSERT INTO `item_mods` VALUES (10646,28,5); -- MAB
+INSERT INTO `item_mods` VALUES (10646,27,-7); -- ENMITY
+INSERT INTO `item_mods` VALUES (10664,1,23); -- SUMMONER'S HORN +2, DEF:23 MP+42 "Blood Pact" ability delay -5 Avatar:"Magic Atk. Bonus"+4 Perpetuation cost -3
+INSERT INTO `item_mods` VALUES (10664,5,42); -- MP
+-- INSERT INTO `item_mods` VALUES (10664,BLOOD PACT ABILITY DELAY,-5);
```

Comment:
MOD_BP_DELAY                  = 357, // stores blood pact delay reduction

or 

    MOD_BP_DELAY_II               = 541, // Blood Pact Delay Reduction II

I don't know which one you want, but yeah.